src/components/Editor/Editor.js

```diff
@@ -13,8 +15,14 @@ import EditorToolbar from './EditorToolbar';
 import Action from '../Button/Action';
 import Body, { remarkable } from '../Story/Body';
 import './Editor.less';
+import { getWordCount } from '../../reducers';
 
 @injectIntl
+@connect(
```

Comment:
I would rather keep Editor as a state-less component, so use connect from its parent and just pass it by props.